packages/paypal-js/src/load-script.ts

```diff
@@ -104,6 +104,13 @@ function validateArguments(options: unknown, PromisePonyfill?: unknown) {
     if (typeof options !== "object" || options === null) {
         throw new Error("Expected an options object.");
     }
+    const { environment } = options as PayPalScriptOptions;
+
+    if (environment && !["production", "sandbox"].includes(environment)) {
```

Comment:
I prefer to remove this validation code for two reasons:
1. The logic in utils.ts will ignore any bad values and will default the production environment.
2. paypal-js is designed to compile to ES3 so using Array.prototype.includes() would be a breaking change. If we really want to keep this, let's replace the usuage of includes() with something that transpiles nicely to ES3 and does not require a polyfill.

This loader is meant to be as small as possible on top of the PayPal JS SDK.

Reply:
I agree with the challenges with `Array.prototype.includes()` . We can definitely use an alternative there 👍 .

About not having validations on input, once production stops the support for sandbox, then this may become a source of errors, if there is any typos in the input options.

Reply:
I see your point. Let's replace the usage of `Array.prototype.includes()` with some old school JS then :)

Reply:
done ✔️